uvicorn/protocols/websockets/wsproto_impl.py

```diff
@@ -131,11 +155,11 @@ def handle_no_connect(self, event):
             (b"connection", b"close"),
```

Comment:
It seems like this `handle_no_connect()` method is not covered by the test suite ...